### ISSUE TITLE
No code write feature?

### ISSUE THREAD

**Issue by vijaybhaskar14** [bridgedxyz/assistant#79]
Is this bridged completely have the no code write feature means no need write code manually for alignments , color backgrounds & layout adjustment  corrections  ?? or we can modify code manually ??

**For Ex : if the design screen have multiple functions it can be work as per the design when we convert using bridged**
coz i had tried XD to Flutter also (actually we can do lot of manual work in XD to flutter ) is this same way to bridged??

**Comment by softmarshmallow** [comment]
1. Layout Engine - interprets layout based on auto layout, alignment
2. Linter Engine - Lints the design before converting to code (tells if design is wrongly configured, or to suggest to use autolayout etc..)
2. Component Engine - (THIS IS WHAT YOUR LOOKING FOR) We provide code override feature for components / instance. when the screen's code being generated, we'll check if the desired node is a component, and if has a overrided code.
3. [Detection](https://github.com/bridgedxyz/reflect-detection) Engine - detects the group of layers if it's a general component such as button, chip, input, avatar, etc..
4. [Reflect](https://github.com/bridgedxyz/reflect-core-ts) Engine
5. [CoLI](https://github.com/bridgedxyz/CoLI) Engine, and various exporter including [flutter-builder](https://github.com/bridgedxyz/flutter-builder)

More details / talks about component.

here, we made a ui to override code for a selected component. https://github.com/bridgedxyz/assistant/pull/72
NOT YET the generation uses the overrided code, this feature is pre-steps of implementing what you've requested.

related
https://github.com/bridgedxyz/assistant/issues/77
https://github.com/bridgedxyz/assistant/issues/63
https://github.com/bridgedxyz/assistant/issues/14


TL;DR The feature you've asked is critical for developer's needs. And we are working on it!

**Comment by vijaybhaskar14** [comment]
Thanks For Your SOPPORT

**Comment by softmarshmallow** [comment]
We'll leave this issue open for the future releases.
